src/renderer/components/helpers/ChatMethods.ts

```diff
@@ -338,11 +338,14 @@ export async function modifyGroup(
 ) {
   const accountId = selectedAccountId()
   log.debug('action - modify group', { chatId, name, image, members })
-  await BackendRemote.rpc.setChatName(accountId, chatId, name)
+
   const chat = await BackendRemote.rpc.getFullChatById(accountId, chatId)
   if (!chat) {
     throw new Error('chat is undefined, this should not happen')
   }
```

Comment:
`await BackendRemote.rpc.getFullChatById(accountId, chatId)` will already throw in that case so this if statement can be removed, probably comes from the time of using the cffi api in desktop where there is not so good error handling (check for 0 as hint there was an error, then fish the error from the core events, sometimes not even that)

getFullChatById can not return `null` or `undefined`, this is it's rust function signature:
```rs
 async fn get_full_chat_by_id(&self, account_id: u32, chat_id: u32) -> Result<FullChat>
```